scripts/misc/identify_missing_scenes.md

```diff
@@ -0,0 +1,58 @@
+# Identifying missing scenes
+
+We needed to identify scenes based on their URI, so that we can find duplicates.
+From the duplicate that is not archived, we need to identify the source Landsat
+scenes. This parent's ID is required so that it can be reprocessed for WOfS
+and FC.
+
+## Merge CSVs
+
+```python
+import pandas as pd
+
+csv1 = pd.read_csv("landsat_archived_2022-01-27.csv")
+csv2 = pd.read_csv("landsat_archived_2022-02-02.csv")
+csv3 = pd.read_csv("landsat_archived_2022-02-09.csv")
+csvs = [csv1, csv2, csv3]
+all = pd.concat(csvs)
+
+fc_scenes = all[all["product"]=="fc_ls"]
+fc_scenes["location"].to_csv("fc.csv", index=False, header=False)
+```
+
+## Identify active duplicate's parent
+
+```python
+import datacube
+dc = datacube.Datacube()
+index = dc.index
+
+def get_sr_id(uri):
+    for dataset in index.datasets.get_datasets_for_location(uri):
+        if not dataset.is_archived:
+            break
+
+    dataset_with_parents = index.datasets.get(dataset.id, include_sources=True)
```

Comment:
is this under for loop?

Reply:
No it's not. But there's a bug in it, really... It was super hacky and fast!